SDK/MobileCenter/Microsoft.Azure.Mobile.Android/MobileCenter.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.Mobile
     /// <summary>
     /// SDK core used to initialize, start and control specific service.
     /// </summary>
-    public static partial class MobileCenter
+    public partial class MobileCenter
```

Comment:
Can you explain?

Need to hide constructor if you do that (be declaring internal one).